Makefile

```diff
@@ -11,6 +10,11 @@ CFLAGS += -Wno-unused-label
 CFLAGS += -g
 LDFLAGS += -lm
 
+# tcc doesn't have <stdnoreturn.h>
+ifeq ($(CC),tcc)
```

Comment:
Look for substring instead of matching the entire string:

```suggestion
# https://stackoverflow.com/a/37741652
ifeq ($(findstring tcc,$(CC)),tcc)
```